diff --git a/node_modules/@types/istanbul-reports/index.d.ts b/node_modules/@types/istanbul-reports/index.d.ts
index ea6d16d..f09abbb 100755
--- a/node_modules/@types/istanbul-reports/index.d.ts
+++ b/node_modules/@types/istanbul-reports/index.d.ts
@@ -5,9 +5,9 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.4
 
-import { Node, ReportBase } from 'istanbul-lib-report';
+import { Node } from 'istanbul-lib-report';
+
 
-export function create<T extends keyof ReportOptions>(name: T, options?: Partial<ReportOptions[T]>): ReportBase;
 
 export interface FileOptions {
     file: string;
